web/lattice_surgery_web/settings_template_test.py

```diff
@@ -0,0 +1,126 @@
+"""
```

Comment:
Maybe this file was committed accidentally?

Reply:
This is actually an alternate settings.py file which is used when render_to_file.py is called. In render_to_file, the django environment is loaded but I needed to to specify full paths for the static files so that it finds the static files such as css and images on your local browser when you debug. Perhaps it should be renamed according to how you use the render to file script as a debugger. Maybe settings_render_to_file.py?

Reply:
changed it to settings_render_to_file.py